src/sass/_style.scss

```diff
@@ -290,22 +289,22 @@ select.form-control {
 
 #password-requirement-left {
   opacity: 1;
-  @extend .x-small;
+  font-size: var(--pgn-typography-font-size-xs) !important;
```

Comment:
It looks like `.x-small` is still listed in css utilities on the docs site https://paragon-openedx.netlify.app/foundations/css-utilities/

Does continuing to use `.x-small` here break things?

Reply:
This change is good as-is (see https://github.com/openedx/frontend-app-authn/pull/1436#discussion_r2079869474 for context)